tests/integration/test_payment_processing.py

```diff
@@ -0,0 +1,132 @@
+import json
+
+import stripe
+from django.urls import reverse
+from djstripe.models import PaymentMethod, Customer
+from djstripe.sync import sync_subscriber
+# from djstripe.core import Price
+from rest_framework.test import APITestCase
+
+from django.contrib.auth import get_user_model
+
+from ckc.stripe.utils.payments import create_checkout_session, create_payment_intent, confirm_payment_intent
+from ckc.stripe.utils.subscriptions import create_price
+
+User = get_user_model()
+
+
+class TestPaymentProcessing(APITestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username="test", password="test")
+        self.client.force_authenticate(user=self.user)
+        return super().setUp()
```

Comment:
let's do a `setUpTestData` (runs only once for a class, re-used between functions)

Can throw `customer, created = Customer.get_or_create(subscriber=self.user)` in it as well

example:

```py
    @classmethod
    def setUpTestData(cls):
        cls.user = User.objects.create_user(username="test", password="test")
        cls.client.force_authenticate(user=self.user)
        cls.customer, cls.created = Customer.get_or_create(subscriber=cls.user)
```

if `cls.client` isn't available, you may have to do that `force_authenticate` step in the `setUp` function instead